scripts/windows-setup.ps1

```diff
@@ -48,13 +48,15 @@ if (!$SkipTools) {
 
         Invoke-WebRequest -UseBasicParsing -Uri 'https://get.scoop.sh' `
             | Invoke-Expression
+        scoop install git go nvm
         scoop bucket add versions
-        scoop install git go nvm python27
+        scoop install python
+        # Install and use latest node 16* version
+        nvm install 16
+        nvm use $(nvm list | Where-Object { $_ -match '16\..+\..+'} | Select-Object -First 1).Trim()
```

Comment:
… `nvm list | Select-String '[16\.[0-9.]+]' | Select-Object -First 1 | ForEach-Object { $_.Matches.Value }`
The `Select-String` part just simplifies the expression.
The `ForEach-Object` part ensures we don't get the "Currently using…" part, if it's active.

Reply:
@mook-as , Modified the command per your suggestion.